bin/rad-issue

```diff
@@ -231,7 +236,8 @@
                _ (put-str! "No selection made."))
         (do
           (put-str! (pretty-table-header))
-          (map (fn [d] (put-str! (first d))) items))))))
+          (map (fn [d] (put-str! (first d))) items)
+          (put-str! (items/summary "issues" issues)))))))
```

Comment:
Could the list function be completely moved to `items.rad` It would be nice in general to extract more code that is shared by both apps and possibly used by new ones too.

Reply:
I will do it in a separate PR.

Reply:
#12 in Radicle issues